code/src/components/Thought.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ThoughtLikes } from './ThoughtLikes';
+
+export const Thought = ({
+  message,
+  hearts,
+  time,
+  id,
+  onLikedThought
+}) => {
+  return (
+    <div className="thought-container">
+      <p key={id} id={id}>
```

Comment:
You don't need to use `key` prop here, we are not performing any loop

Reply:
Removed.